doc(storage-network): update IP usage (#655)
harvester/docs: docs/advanced/storagenetwork.md

- IP range for Storage Network
	- IP range should not conflict or overlap with Kubernetes cluster networks(`10.42.0.0/16`, `10.43.0.0/16`, `10.52.0.0/16` and `10.53.0.0/16` are reserved).
	- IP range should be in IPv4 CIDR format and Longhorn pods use Storage Network as follows:

>     - `instance-manger` pods: Starting with **v1.5.0**, [Longhron instance managers are consolidated](https://longhorn.io/docs/1.5.0/deploy/important-notes/#instance-managers-consolidated). Previous engine/replica instance managers are now deprecated. These require 1 IP per node. During an upgrade, two versions of these pods will exist (old and new), and the old version will be deleted once the upgrade is successful.

Reviewer: ```suggestion
    - `instance-manager` pods: Longhorn Instance Manager components were [consolidated in Longhorn v1.5.0](https://longhorn.io/docs/1.5.0/deploy/important-notes/#instance-managers-consolidated). The Engine Instance Manager and Replica Instance Manager are now deprecated. One IP is required for each node. During an upgrade, two versions of these pods will exist (old and new), and the old version will be deleted once the upgrade is successful.
```

    - `backing-image-ds` pods: These are employed to process on-the-fly uploads and downloads of backing image data sources. These pods will be removed once the image uploads or downloads are completed.
    - `backing-image-manager` pods: 1 IP per disk, similar to the instance manager pods. Two versions of these will coexist during an upgrade, and the old ones will be removed after the upgrade is completed.

>     - The required number of IPs is calculated using a simple formula: `Required Number of IPs = Number of Nodes * 2 + Number of Disks * 2 + Number of Images to Download/Upload`
> 	- For example, if your cluster has five nodes, each node has two disks, and ten images will be uploaded simultaneously, the IP range should be greater than or equal to `/26` (`5 * 2 + 5 * 2 * 2 + 10 = 40`).

Reviewer: ```suggestion
    - The required number of IPs is calculated using a simple formula: `Required Number of IPs = (Number of Nodes * 2) + (Number of Disks * 2) + Number of Images to Download/Upload`
	- Example: If a cluster has five nodes with two disks each, and ten images will be uploaded simultaneously, the IP range should be greater than or equal to `/26` (`(5 * 2) + (5 * 2) + 10 = 30`).
```

Reviewer: I added parentheses to the formula and example to make reading easier. The original example has an extra `* 2`, which doesn't match the formula. Please check.
